server/utils/applicationUtils.test.ts

```diff
@@ -32,4 +37,71 @@ describe('applicationUtils', () => {
       )
     })
   })
+
+  describe('getService', () => {
```

Comment:
Excellent to see the code and tests going into the same commit, love to see it 👍